db.inc.php

```diff
@@ -41,7 +42,14 @@ public function prepareInsert($tableName)
         return TRUE;
```

Comment:
This always returns TRUE even if you fail to connect to the database. It should return false on failure.